src/datagen/pii/privy/requirements.in

```diff
@@ -3,6 +3,7 @@
 alive-progress==3.1.4
 baluhn==0.1.2
 dicttoxml==1.7.4
+exceptiongroup==1.1.2
```

Comment:
since it's an indirect dependency, this shouldn't be necessary.
think of `requirements.in` as direct deps and `requirements.bazel.txt` as the full specified lockfile that includes indirect deps.

Reply:
Fixed - thanks!